nuxeo-features/nuxeo-automation/nuxeo-automation-features/src/test/java/org/nuxeo/ecm/automation/core/operations/users/TestSuggestUserEntries.java

```diff
@@ -276,4 +276,95 @@ public void testMixedEntries() throws Exception {
         }
     }
 
+    @Test
+    public void testSubGroupEntries() throws Exception {
+
```

Comment:
why remove the schema showing the group hierarchy?

Reply:
I thought it wasn't necessary to understand the test

Reply:
OK, not necessary, but nice to have a quick view.
As you wish!